src/solve.jl

```diff
@@ -539,6 +539,10 @@ function __init__()
                 maxiters = convert(Int, maxiters)
             end
 
+            if splits == nothing
```

Comment:
For this to work you would need to set default value of `splits` kwarg to `nothing`

Reply:
Thanks for pointing it out, I forgot about it. Changed it now.